x-pack/plugins/apm/ftr_e2e/cypress/support/commands.ts

```diff
@@ -18,10 +18,12 @@ Cypress.Commands.add('loginAsPowerUser', () => {
 Cypress.Commands.add(
   'loginAs',
   ({ username, password }: { username: string; password: string }) => {
-    cy.log(`Logging in as ${username}`);
+    // cy.session(username, () => {
```

Comment:
Should some of this be cleaned up?

Reply:
This looks like the same on main too @jbudz